src/main/java/dev/emi/trinkets/api/PlayerTrinketComponent.java

```diff
@@ -69,7 +69,7 @@ public boolean equip(ItemStack stack){
 			if(stack.getItem() == Items.ELYTRA){
 				for(SlotGroup group: TrinketSlots.slotGroups){
 					for(Slot slot: group.slots){
-						if(group.getName().equals("chest") && slot.getName() == "cape"){
+						if(group.getName().equals(SlotGroups.CHEST) && slot.getName() == Slots.CAPE){
```

Comment:
This has inconsistent comparison still.

Reply:
I considered this out of scope for my changes, I do not fix string comparisons elsewhere.

Reply:
valid